### PR TITLE
Next template as default

### PR DESCRIPTION
**Pull request by zBritva**

### CODE REVIEW

File: templates/visuals/rvisual/pbiviz.json [hunk 1]
```diff
@@ -0,0 +1,2 @@
+{
```
Comment (ignatvilesov) [inline templates/visuals/rvisual/pbiviz.json]: Why we need an empty file?

File: spec/e2e/pbivizNewSpec.js [hunk 2]
```diff
@@ -58,7 +58,7 @@ describe("E2E - pbiviz new", () => {
         let template = 'default';
         let visualPath = path.join(tempPath, visualName);
 
-        FileSystem.runPbiviz('new', visualName);
+        FileSystem.runPbiviz('new', visualName, ' -t default1');
```
Comment (ignatvilesov) [inline spec/e2e/pbivizNewSpec.js]: Why do we need two templates now? Can't we force all of new developers to use ES2015.
I think it makes sense not to use old modules anymore.

File: templates/visuals/rvisual/pbiviz.json (hunk 1 above)
Comment (zBritva) [reply templates/visuals/rvisual/pbiviz.json]: If pbiviz wasn't specified, tools generates pbiviz with externalJS files.

File: spec/e2e/pbivizNewSpec.js (hunk 2 above)
Comment (zBritva) [reply spec/e2e/pbivizNewSpec.js]: We can't break backward compatibility. Old templates uses in UTs

File: templates/visuals/rvisual/pbiviz.json (hunk 1 above)
Comment (ignatvilesov) [reply templates/visuals/rvisual/pbiviz.json]: but pbiviz.json should exist as it declares metadata such as author name and email.

File: spec/e2e/pbivizNewSpec.js (hunk 2 above)
Comment (ignatvilesov) [reply spec/e2e/pbivizNewSpec.js]: ok

File: templates/visuals/rvisual/pbiviz.json (hunk 1 above)
Comment (zBritva) [reply templates/visuals/rvisual/pbiviz.json]: tools gets externalJS files from this files, other metadata comes from the common template for pbiviz.json

**Pull request merged**